autogpt_platform/backend/backend/blocks/reddit.py

```diff
@@ -48,25 +48,25 @@ def get_praw(creds: RedditCredentials) -> praw.Reddit:
 
 class GetRedditPostsBlock(Block):
     class Input(BlockSchema):
-        subreddit: str = Field(description="Subreddit name")
-        creds: RedditCredentials = Field(
+        subreddit: str = SchemaField(description="Subreddit name")
+        creds: RedditCredentials = SchemaField(
             description="Reddit credentials",
             default=RedditCredentials(),
         )
-        last_minutes: int | None = Field(
+        last_minutes: int | None = SchemaField(
             description="Post time to stop minutes ago while fetching posts",
             default=None,
         )
-        last_post: str | None = Field(
+        last_post: str | None = SchemaField(
             description="Post ID to stop when reached while fetching posts",
             default=None,
         )
-        post_limit: int | None = Field(
+        post_limit: int | None = SchemaField(
             description="Number of posts to fetch", default=10
```

Comment:
should these be `| None` since they have a default

Reply:
Likely it was None as the default at first. I'll make the block code change outside of this refactor PR.